libs/causality/src/lib/CausalAnalysisDashboard/Controls/CausalAnalysisView/CausalIndividualView/CausalIndividualChart.tsx

```diff
@@ -458,6 +458,40 @@ export class CausalIndividualChart extends React.PureComponent<
         trace.y = rawY;
       }
     }
+    if (
+      this.context.jointDataset.datasetMetaData?.featureMetaData !== undefined
+    ) {
+      const identityFeatureName =
+        this.context.jointDataset.datasetMetaData.featureMetaData
+          ?.identity_feature_name;
+
+      if (identityFeatureName !== undefined) {
```

Comment:
same suggestion, maybe use:
```!identityFeatureName```
instead of comparing to undefined

Reply:
same below for jointDatasetFeatureName

Reply:
Done